### PR TITLE
[frio] Flip replyto link

### LINKED ISSUE #13520: [FRIO] Size of font for split post does not fit
- [x] I have searched open and closed issues for duplicates

### Bug Description
The font size for the "replay to" and "xy has shared the" are different sizes.

In the stable version 

```
p.wall-item-announce, .media .time, .media .shared-time, .media .delivery, .media .location, .media .location a {
  font-size: 11px;
  color: #000000;
}
```
pulled. In the RC it does not work for the reference to a shared post.

### Platform Info

Friendica Version: RC

Friendica Source: git

PHP version: 8.1

SQL version: 10.5

### LINKED ISSUE #13524: [Frio] Suggestion to place the new "Reply to" function at Favicon
First of all, thank you for this wonderful and very useful feature "in reply to".
However, I would like to suggest that the function be placed somewhere else, because

1. the number of references to a normal post or a reaction to a post keeps increasing.
2. the clarity suffers as a result
3. I, as a user, am overwhelmed by the classification of this information.

I have looked at how Bluesky, for example, do it. Here, the jump mark is placed discreetly next to the user's favicon.

grafik

grafik

Since a reference and information within a thread and the sharing of a post are two fundamentally different pieces of information, it would, in my view, be more comprehensible for the user if the sharing continues to be above a post and the link mark to a content moves to the user favicon as a personal reference.

grafik

### PR DESCRIPTION
**Pull request by xundeenergie**
Resolves #13524 and #13520

### CODE REVIEW

**Comment by Raroun** [comment]
Just a question:
Is it possible that "reply to" shows up on transifex to be translated? :)

**Comment by xundeenergie** [comment]
> Just a question: Is it possible that "reply to" shows up on transifex to be translated? :)

It's made translateable, as all other text too, for tranisflex.

**Comment by xundeenergie** [comment]
The new text for comments without parents:

on small screen:
Bildschirmfoto vom 2023-10-09 20-05-49

and on wide screen
Bildschirmfoto vom 2023-10-09 20-06-33

**Comment by xundeenergie** [comment]
Btw... this is such a comment
https://soc.schuerz.at/display/67d0f1d0-abb2c986d4e1ba42-5ddcaaba

And i don't know, why messages.po_check failed...

i merged 2023.09-rc into this branch, run bin/run_xgettext.sh and committed all.
Then i pushed to github.

What's wrong?

**Comment by MrPetovan** [comment]
Maybe another pull request was merged in the meantime, which triggers the merge conflict? In doubt, do the merge + script execution again.

**Comment by xundeenergie** [comment]
I've done the merge and commit again.

File: src/Object/Post.php [hunk 1]
```diff
@@ -499,10 +499,13 @@ public function getTemplateData(array $conv_responses, string $formSecurityToken
 
 		$parent_guid     = $thread_parent[$item['thr-parent-id']]['guid'] ?? '';
 		$parent_username = $thread_parent[$item['thr-parent-id']]['name'] ?? '';
+		$parent_unknown  = $parent_username ? '' : DI::l10n()->t('parent unknown');
 
 		$tmp_item = [
 			'parentguid'      => $parent_guid,
-			'isreplyto'       => DI::l10n()->t('in reply to %s', $parent_username),
+			'isreplyto'       => DI::l10n()->t('is reply to %s', $parent_username),
```
Comment (MrPetovan) [inline src/Object/Post.php]: Please keep the `in reply to` translation. Additionally, please rename all `isreplyto` variables and array keys to `inreplyto` to match the translation string change.

File: src/Object/Post.php [hunk 2]
```diff
@@ -499,10 +499,13 @@ public function getTemplateData(array $conv_responses, string $formSecurityToken
...
+			'isunknown'       => $parent_unknown,
+			'isunknown_label' => DI::l10n()->t('parent is probably private or not federated'),
```
Comment (MrPetovan) [inline src/Object/Post.php]:
```suggestion
			'isunknown_label' => DI::l10n()->t('Parent is probably private or not federated.'),
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Object/Post.php (hunk 1 above)
Comment (xundeenergie) [reply src/Object/Post.php]: Ok.
But why? 
It sounds strange to my ears...

Comment (xundeenergie) [reply src/Object/Post.php]: i changed it and all occurances

Comment (MrPetovan) [reply src/Object/Post.php]: Thanks, the (American-)English expression is "in reply to" as evidenced in the Mail header `In-Reply-To`. You could also say "This is a reply to" but it's longer and requires a subject that "in reply to" naturally implies.

File: src/Object/Post.php [hunk 3]
```diff
@@ -499,10 +499,13 @@ public function getTemplateData(array $conv_responses, string $formSecurityToken
 
 		$parent_guid     = $thread_parent[$item['thr-parent-id']]['guid'] ?? '';
 		$parent_username = $thread_parent[$item['thr-parent-id']]['name'] ?? '';
+		$parent_unknown  = $parent_username ? '' : DI::l10n()->t('parent unknown');
```
Comment (MrPetovan) [inline src/Object/Post.php]:
```suggestion
		$parent_unknown  = $parent_username ? '' : DI::l10n()->t('Unknown parent');
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Comment by MrPetovan** [comment]
Thank you for the change, you will have to regenerate the messages.po file.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**